helm/flowforge/README.md

```diff
@@ -119,6 +119,16 @@ Enables FlowForge Telemetry
  - `forge.privateCA.configMapName` name of ConfigMap to store the CA Cert bundle (default `ff-ca-certs`)
  - `forge.privateCA.certs` base64 encoded CA certificate PEM bundle of trusted certificates. This needs to be generated without line breaks e.g. `base64 -w 0 certs.pem` (default not set)
  
+ ### Rate Limiting
+
+ - `forge.rate_limits.enabled` (default `false`)
+ - `forge.rate_limits.global` (default `false`)
+ - `forge.rate_limits.timeWindow` Time in milliseconds to evaluate requests over (default 60000)
+ - `forge.rate_limits.max` Max requests per timeWindow (default: 1000)
+ - `forge.rate_limits.maxAnonymous` Max anonymous requests per timeWindow (default: 10)
```

Comment:
Are these public defaults or our cloud solution defaults Ben?

If general release defaults, these should be...

```suggestion
 - `forge.rate_limits.enabled` (default `false`)
 - `# forge.rate_limits.global` (default `true`)
 - `# forge.rate_limits.timeWindow` Time in milliseconds to evaluate requests over (default 60000)
 - `# forge.rate_limits.max` Max requests per timeWindow (default: 1000)
 - `# forge.rate_limits.maxAnonymous` Max anonymous requests per timeWindow (optional)
```


From the docs...
<html>
<body>
<!--StartFragment-->
flag | desc
-- | --
rate_limits.enabled | Enables rate limiting. Default: false
rate_limits.global | Enables rate limiting for all routes. Default: true (defaults to all routes being rate limited)
rate_limits.timeWindow | The time window in which requests are counted. Default: 60000 (1 minute)
rate_limits.max | The maximum number of requests allowed in the time window. Default: 1000
rate_limits.maxAnonymous | The maximum number of requests allowed in the time window for anonymous users. Default: not configured (defaults to rate_limits.max)

<!--EndFragment-->
</body>
</html>

Reply:
We do not include comments in the docs!

Reply:
The value in the docs MUST match the values in the values.yml file. I will fix the global value

Reply:
Actually, no, the docs already match the values.

I took the defaults from the forge PR, so they are the actual defaults, we haven't set the production values (and these do NOT match what what you've asked for on staging)

Reply:
Im lost off Ben.  (probably my fault)

For general release, the conclusion was that `forge.rate_limits.global` would be `true` by default

Which docs show this as `false`?

The merged FF docs show it as true - https://github.com/flowforge/flowforge/blob/main/docs/install/configuration.md#rate-limiting-configuration